src/app/components/bot/BotPreview.js

```diff
@@ -154,102 +269,165 @@ const BotPreview = ({ me, team }) => {
-            <h6>Bot Preview</h6>{' '}
-            <sup className={styles.beta}>BETA</sup>
-            { messageHistory.length ?
-              <>
-                {' - '}
-                <button className={styles.buttonAsLink} onClick={() => setDialogOpen(true)}>reset chat</button>
-              </> : null }
-            <ConfirmProceedDialog
-              body="Are you sure you would like to clear this chat? All messages will be removed"
-              open={dialogOpen}
-              proceedLabel="Yes, clear chat"
-              title="Reset Chat"
-              onCancel={() => setDialogOpen(false)}
-              onProceed={resetHistory}
+    <>
+      { settingsHaveChanged && (
+        <Alert
+          banner
+          buttonLabel={
+            <FormattedMessage
+              defaultMessage="Apply Changes to Live Bot"
+              description="Label for the button to apply changes to the bot settings."
+              id="botPreview.applyChangesButton"
+            />
+          }
+          content={
+            <FormattedMessage
+              defaultMessage="Use the Bot Preview below to see the results of the changes you have made to the settings. When ready, apply changes to publish the settings to the Live Tipline Bot for this workspace."
+              description="Description of the alert message displayed on the bot preview page to warn about unsaved changes."
+              id="botPreview.changesAlertContent"
             />
+          }
+          extraActions={
+            <ButtonMain
+              label="Reset All Changes"
+              size="small"
+              theme="text"
+              variant="outlined"
+              onClick={() => setConfirmBeforeReset(true)}
+            />
+          }
+          title={
+            <FormattedMessage
+              defaultMessage="Changes Made to Bot Settings"
+              description="Title of the alert message displayed on the bot preview page to warn about unsaved changes."
+              id="botPreview.changesAlertTitle"
+            />
+          }
+          variant="success"
+          onButtonClick={() => setConfirmBeforeCommit(true)}
+        />
+      )}
+      { confirmBeforeCommit && (
+        <ConfirmProceedDialog
+          body="If you proceed, the changes you have made to the bot settings will be applied to the live bot for this workspace."
+          open={confirmBeforeCommit}
+          proceedLabel="Apply Changes"
+          title="Apply Changes to Live Bot"
+          onCancel={() => setConfirmBeforeCommit(false)}
+          onProceed={() => {
+            setConfirmBeforeCommit(false);
+            saveAllSettings();
+          }}
+        />
+      )}
+      { confirmBeforeReset && (
+        <ConfirmProceedDialog
+          body="If you proceed, the changes you have made to the bot settings will be discarded."
+          open={confirmBeforeReset}
+          proceedLabel="Discard Changes"
+          title="Discard Changes"
```

Comment:
add question marks to the two confirm dialog titles `Discard Changes` => `Discard Changes?`